shell/browser/native_window_views.cc

```diff
@@ -1378,6 +1393,21 @@ bool NativeWindowViews::IsMenuBarVisible() {
   return root_view_->IsMenuBarVisible();
 }
 
+void NativeWindowViews::SetBackgroundMaterial(const std::string& material) {
+#if BUILDFLAG(IS_WIN)
+  // DWMWA_USE_HOSTBACKDROPBRUSH is only supported on Windows 11 22H2 and up.
+  if (base::win::GetVersion() < base::win::Version::WIN11_22H2)
+    return;
+
+  DWM_SYSTEMBACKDROP_TYPE backdrop_type = GetBackdropFromString(material);
+  HRESULT result =
+      DwmSetWindowAttribute(GetAcceleratedWidget(), DWMWA_SYSTEMBACKDROP_TYPE,
+                            &backdrop_type, sizeof(backdrop_type));
+  if (FAILED(result))
+    LOG(WARNING) << "Failed to set background material to " << material;
```

Comment:
User facing error instead of LOG ?

Reply:
I'm also open to that, but we don't have a clear convention there either really 🤔 for dark mode for example we don't:
https://github.com/electron/electron/blob/5d120359f6f6adf730455189d099b0a63a0e5cc9/shell/browser/win/dark_mode.cc#L57

Beyond that - we don't really get a good error message from the call sadly, so all we'd really be able to say is "it failed"

Reply:
Because we set this in the constructor we can't safely have a user facing error afaik anyway.  I think the LOG is fine here.  We shouldn't fail to construct a window just because the DWM call got lost